requirements.txt

```
@@ -1,3 +1,4 @@
-Flask==1.0.2
+Flask==2.0.1
```

Thanks for validating the new version! But I'd like to keep this change into a different PR and this upgrade will be addressed on #26 
```suggestion
Flask==1.0.2
```

---

Sounds good, I reverted it :)